includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -139,7 +144,8 @@ private function getSearchData() {
 			'type' => 'search',
 			'results' => [
 				'url' => $searchUrl,
-				'param-name' => $this->product === static::PRODUCT_FANDOMS ? 's' : 'query'
+				'param-name' => $this->product === static::PRODUCT_FANDOMS ? 's' : 'query',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "param-name" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)